jdi-light-html/src/main/java/com/epam/jdi/light/ui/html/elements/complex/Menu2D.java

```diff
@@ -0,0 +1,279 @@
+package com.epam.jdi.light.ui.html.elements.complex;
+
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.*;
+import com.epam.jdi.light.ui.html.elements.annotations.*;
+import com.epam.jdi.light.ui.html.elements.enums.MenuBehaviour;
+import com.epam.jdi.tools.func.JAction1;
+import org.openqa.selenium.By;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.*;
+
+import static com.epam.jdi.light.common.Exceptions.*;
+import static com.epam.jdi.light.driver.WebDriverByUtils.*;
+import static com.epam.jdi.light.elements.init.UIFactory.*;
+import static com.epam.jdi.light.elements.pageobjects.annotations.objects.FillFromAnnotationRules.*;
+import static com.epam.jdi.tools.LinqUtils.*;
+import static com.epam.jdi.tools.PrintUtils.*;
+import static java.lang.String.*;
+import static java.util.Arrays.*;
+import static org.apache.commons.lang3.StringUtils.*;
+
+/**
+ * Created by Roman Iovlev on 02.05.2020
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class Menu2D extends Selector implements ISetup {
+    protected List<String> locators = new ArrayList<>();
+    protected String separator = ">";
+    protected String printFormat = "%s[%s]";
+    protected String printSeparator = ";";
+    protected JAction1<UIElement> pathAction = UIElement::click;
+    protected JAction1<UIElement> lastAction = UIElement::click;
+    protected boolean inheritLocators = true;
+
+    public Menu2D() { }
+    public Menu2D(String... locators) {
+        this.locators = asList(locators);
+    }
+
+    protected void setSeparator(String separator) {
+        this.separator = separator;
+    }
+    protected void setPrint(String printFormat, String printSeparator) {
+        this.printFormat = printFormat;
+        this.printSeparator = printSeparator;
+    }
+    protected void setActions(JAction1<UIElement> pathAction, JAction1<UIElement> lastAction) {
+        this.pathAction = pathAction;
+        this.lastAction = lastAction;
+    }
+    protected void setActions(MenuBehaviour behaviour) {
+        switch (behaviour) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)